docs/README.md

```diff
@@ -26,92 +26,183 @@
+- [Framework-Specific Optimization](user_guide/optimization.md#framework-specific-optimization)
+  - [ONNX-TensorRT](user_guide/optimization.md#onnx-with-tensorrt-optimization-ort-trt)
+  - [ONNX-OpenVINO](user_guide/optimization.md#onnx-with-openvino-optimization)
+  - [TensorFlow-TensorRT](user_guide/optimization.md#tensorflow-with-tensorrt-optimization-tf-trt)
+  - [TensorFlow-Mixed-Precision](user_guide/optimization.md#tensorflow-automatic-fp16-optimization)
+- [NUMA Optimization](user_guide/optimization.md#numa-optimization)
+
+#### Scheduling and Batching
+
+Triton supports batching individual inference requests to improve compute resource utilization. This is extremely important as individual queries will not saturate GPU resources thus not leveraging the parallelism provided by GPUs to its extent. Learn more about Triton's [Batcher and Scheduler](user_guide/model_configuration.md#scheduling-and-batching).  
+- [Default Scheduler - Non-Batching](user_guide/model_configuration.md#default-scheduler)
+- [Dynamic Batcher](user_guide/model_configuration.md#dynamic-batcher)
+  - [How to Configure Dynamic Batcher](user_guide/model_configuration.md#recommended-configuration-process)
+    - [Delayed Batching](user_guide/model_configuration.md#delayed-batching)
+    - [Preferred Batch Size](user_guide/model_configuration.md#preferred-batch-sizes)
+  - [Preserving Request Ordering](user_guide/model_configuration.md#preserve-ordering)
+  - [Priority Levels](user_guide/model_configuration.md#priority-levels)
+  - [Queuing Policies](user_guide/model_configuration.md#queue-policy)
+  - [Ragged Batching](user_guide/ragged_batching.md)
+- [Sequence Batcher](user_guide/model_configuration.md#sequence-batcher)
+  - [Stateful Models](user_guide/architecture.md#stateful-models)
+  - [Control Inputs](user_guide/architecture.md#control-inputs)
+  - [Implicit State - Stateful Inference Using a Stateless Model](user_guide/architecture.md#implicit-state-management)
+  - [Sequence Scheduling Strategies](user_guide/architecture.md#scheduling-strateties)
+    - [Direct](user_guide/architecture.md#direct)
+    - [Oldest](user_guide/architecture.md#oldest)
+
+#### Rate Limiter
+Rate limiter manages the rate at which requests are scheduled on model instances by Triton. The rate limiter operates across all models loaded in Triton to allow cross-model prioritization. [Learn more](user_guide/rate_limiter.md).
+
+#### Model Warmup
+For a few of the Backends(check [Additional Resources](README.md#resources)) some or all of intialization is deffered till the first inference request is received, the benefit is resource conservation but comes with the downside of the initial requests getting processed slower than expected. Users can pre-"warm up" the model by instructing Triton to intialize the model. [Learn more](user_guide/model_configuration.md#model-warmup). 
```

Comment:
Nit: There should be a space before the parenthesis, but I think readability would be improved if Additional Resources was moved elsewhere. I'm not sure the context here, but perhaps adding context and putting it at the end of the paragraph would be good. First sentence is a little long too, so might be good to chop it up. And "User can warm up the model" reads a little more clearly to me due to the lack of quotes.